### PR TITLE
Feature/cli adduser

### PR DESCRIPTION
**Pull request by oererenn**

### CODE REVIEW

**Review by michaelbahchevanov — DISMISSED** [review]

**Review by larsverp — DISMISSED** [review]
Nice :)

File: GraphQL_Express_Server/CLI/validation.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
+ * @description Check the input of email
+ * @param {Email} input 
+ * @returns true if the input is correct otherwise returns the error message
+ */
+const validateEmail = (input) => {
+    const re = /(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])/
```
Comment (BondarencoM) [inline GraphQL_Express_Server/CLI/validation.js]: I'll trust you with this one

**Review by AleksandarGT — DISMISSED** [review]

**Review by BondarencoM — DISMISSED** [review]

**Review by larsverp — APPROVED** [review]
Nice :)

**Review by BondarencoM — APPROVED** [review]

**Review by MolotovHammr — APPROVED** [review]
Great work!

**Review by AleksandarGT — APPROVED** [review]

**Review by michaelbahchevanov — APPROVED** [review]

**Pull request merged**